fetch.bzl

```diff
@@ -79,6 +79,16 @@ def fetch_images():
         tag = "latest",
     )
 
+    oci_pull(
+        name = "ubuntu",
+        image = "index.docker.io/library/ubuntu",
```

Comment:
nit: at HEAD this could just be `image = "ubuntu"`

Reply:
done